coordinator-server/event-client/actionSniffer.js

```diff
@@ -1,12 +1,30 @@
-const AllowedEvents = ['click', 'doubleclick', 'keyup', 'keydown', 'contextmenu', 'scroll', 'mouseover', 'resize'];
+const AllowedEvents = ['click', 'doubleclick', 'keyup', 'keypress', 'keydown', 'contextmenu', 'scroll', 'mouseover', 'resize'];
 const DebounceEvents = ['scroll', 'mouseover', 'resize'];
```

Comment:
Can we also clean up everything related to `debounce`d events? Like this array, the `debounce()` function, etc.